src/main/liquibase/liquibase-ciel-data.xml

```diff
@@ -49,8 +47,8 @@
 	</changeSet>
 
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
-		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />
+		<comment>Adding ciel data</comment>
+		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" ></sqlFile>
```

Comment:
Please change this line back to `<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />`